### DISCUSSION TITLE
Feature: Second animation for resources aka regrowth

### DISCUSSION THREAD

**Discussion by Zano742** [AscensionGameDev/Intersect-Engine#1767] [Ideas]
Adding a second animation for resources, there's the initial one i.e. breaking the resource. 

Adding an animation for a resource "regrowing" would allow more customization of various resources by adding more types of customization.

This enhancement idea is not a duplicate to the best of my knowledge.

**Comment by LunamOdium** [comment]
A better enhancement would be the allowance of turning one resource into another.
E.G: You have a Tree resource and a Stump resource, when you cut the tree down, turn the tree resource into the stump resource so you can now harvest the stump if you wish. Allowing this to work both ways will allow a regrow effect. Turn a sapling resource into a tree resource after certain amount of time, etc(could be done with common events)

Edit: add in the ability to spawn resources using events and this would allow for an easily implemented farming system as well.

**Reply by WeylonSantana** [reply]
> A better enhancement would be the allowance of turning one resource into another. E.G: You have a Tree resource and a Stump resource, when you cut the tree down, turn the tree resource into the stump resource so you can now harvest the stump if you wish. Allowing this to work both ways will allow a regrow effect. Turn a sapling resource into a tree resource after certain amount of time, etc(could be done with common events)
> 
> Edit: add in the ability to spawn resources using events and this would allow for an easily implemented farming system as well.

This sounds like another feature that has nothing to do with the one mentioned above, it would be better if you created another thread by copying and pasting everything mentioned
